include/libjungle/db_config.h

```diff
@@ -605,6 +605,13 @@ class GlobalConfig {
      */
     size_t numFlusherThreads;
 
+    /**
+     * Create a dedicated flusher for log store sync when log store and
+     * normal data store are used. Only effective when
+     * `numFlusherThreads` >= 2.
+     */
+    bool dedicatedSyncOnlyFlusher;
```

Comment:
Let's change the config name, given that the flusher with this will handle the requests by `flushLogsAsync`, no matter what the request is (sync, flush, purge, ...).

I think `dedicatedFlusherForAsyncReqs` will be better.

Reply:
Fixed.